### DISCUSSION TITLE
Access sensor data

### DISCUSSION THREAD

**Discussion by aasjunior** [zepp-health/zeppos-forum#157] [Q&A] [answered]
Could someone tell me if it is possible to capture sensor data and send it to a MongoDB database?, and if it is possible to develop an interface that the user sends a message that is also stored in this database.

**Comment by MyoungXUE** [comment, marked as answer]
Hello, please refer to the Zepp OS Mini Program the overall architecture of https://docs.zepp.com/docs/guides/architecture/arc/

In the Device App, we use the API related to Sensor to obtain data. The Device App communicates with Zepp through Bluetooth, and the sensor data can be uploaded to the server by using the fetchAPI network capability of Zepp.
